content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,186 @@
+---
+reviewers:
+- freehan
+title: EndpointSlice
+feature:
+  title: EndpointSlice
+  description: >
+    Pelacakan _endpoint_ jaringan yang _scalable_ pada klaster Kubernetes.
+
+content_template: templates/concept
+weight: 15
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+EndpointSlice menyediakan sebuah cara yang mudah untuk melacak _endpoint_ jaringan dalam sebuah
+klaster Kubernetes. EndpointSlice memberikan alternatif yang lebih _scalable_ dan lebih dapat diperluas daripada Endpoint.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Motivasi
+
+Endpoint API telah menyediakan sebuah cara yang mudah dan sederhana untuk
+melacak _endpoint_ jaringan pada Kubernetes. Sayangnya, seiring dengan besarnya klaster Kubernetes
+dan Service, batasan-batasan yang dimiliki API tersebut semakin terlihat.
+Terutama, hal tersebut termasuk kendala-kendala mengenai proses _scaling_ _endpoint_ jaringan
+dalam jumlah yang besar.
+
+Karena semua _endpoint_ jaringan untuk sebuah Service disimpan dalam satu sumber daya 
+Endpoint, sumber daya tersebut dapat menjadi cukup besar. Hal itu dapat mempengaruhi performa
+dari komponen-komponen Kubernetes (terutama _master control plane_) dan menyebabkan
+lalu lintas jaringan dan pemrosesan yang cukup besar ketika Endpoint berubah.
+EndpointSlice membantu kamu menghindari masalah-masalah tersebut dan juga menyediakan _platform_
+yang dapat diperluas untuk fitur-fitur tambahan seperti _topological routing_.
+
+## Sumber daya EndpointSlice {#endpointslice-resource}
+
+Pada Kubernetes, sebuah EndpointSlice memiliki referensi-referensi terhadap serangkaian _endpoint_
+jaringan. _Controller_ EndpointSlice secara otomatis membuat EndpointSlice
+untuk sebuah Kubernetes Service ketika sebuah {{< glossary_tooltip text="Selector"
+term_id="selector" >}} dituliskan. EndpointSlice tersebut akan memiliki
+referensi-referensi menuju Pod manapun yang cocok dengan Selector pada Service. EndpointSlice mengelompokkan
```

Comment:
```suggestion
untuk sebuah Kubernetes Service ketika sebuah {{< glossary_tooltip text="selector"
term_id="selector" >}} dituliskan. EndpointSlice tersebut akan memiliki
referensi-referensi menuju Pod manapun yang cocok dengan selector pada Service. EndpointSlice mengelompokkan
```

“selector” is correct; “Selector” suggests that there is a Kubernetes resource named Selector (there isn't).

Reply:
I agree now that you mentioned it. What do you think @wahyuoi?

Reply:
:+1: 
please edit line 131 & 132 too for this case

Reply:
Done that in https://github.com/kubernetes/website/pull/20166/commits/76007160c8037d6a418142a7be8a7dc3155d58fb